server/src/test/java/org/opensearch/action/bulk/TransportBulkActionIngestTests.java

```diff
@@ -170,6 +170,7 @@ class TestTransportBulkAction extends TransportBulkAction {
                     SETTINGS,
                     new ClusterService(SETTINGS, new ClusterSettings(SETTINGS, ClusterSettings.BUILT_IN_CLUSTER_SETTINGS), null)
                 ),
+                null,
```

Comment:
Why are we passing null IndicesService here? Is the code not invoked for these tests?

Reply:
I've provided `mock(...)` wherever we're accessing `IndicesService`.

Reply:
Should we do verifications on the mock as well based on usage?